users/admin.py

```diff
@@ -38,6 +43,115 @@ class LanguagesSpokenAtHomeInline(admin.TabularInline):
     verbose_name_plural = _("Languages spoken at home")
 
 
+def _generate_children_event_history_markdown(modeladmin, request, queryset):
+    if queryset.count() != 1:
+        modeladmin.message_user(
+            request,
+            "Only 1 guardian can be selected for "
+            "'Generate authorization server is changing notification text' action.",
+        )
+        return
+    guardian = queryset.first()
+    return (
+        guardian,
+        AuthServiceNotificationService.generate_children_event_history_markdown(
+            guardian=guardian
+        ).replace("\n", "<br/>"),
+    )
+
+
+@admin.action(description="Generate children event history markdown")
+def generate_children_event_history_markdown(modeladmin, request, queryset):
```

Comment:
Hmm... I was about to say [PEP 8](https://peps.python.org/pep-0008/) and `modeladmin` → `model_admin` and `queryset` → `query_set` because the classes are [ModelAdmin](https://docs.djangoproject.com/en/3.2/ref/contrib/admin/#django.contrib.admin.ModelAdmin) and [QuerySet](https://docs.djangoproject.com/en/3.2/ref/models/querysets/#django.db.models.query.QuerySet) and those parameters are probably instances of those classes. But to my surprise Django's [documentation](https://docs.djangoproject.com/en/3.2/ref/contrib/admin/actions/#writing-action-functions) uses this naming convention in its example:
```python
def make_published(modeladmin, request, queryset):
    queryset.update(status='p')
```

So this is acceptable given that Django is using this in its own documentation and PEP 8's points:

1. [Naming conventions](https://peps.python.org/pep-0008/#naming-conventions):
   - > New modules and packages (including third party frameworks) should be written to these standards, but where an existing library has a different style, internal consistency is preferred.
2. [Function and method arguments](https://peps.python.org/pep-0008/#function-and-method-arguments) section in PEP 8 doesn't actually specify naming convention which I find a bit peculiar

Reply:
I won't change the parameter names here since those are decided by the Django framework (or any other 3rd party whose functions we are extending here). In my opinion it would be worse if the variables are renamed by us. Foe example the "queryset" is used all around the django ecosystem.